src/components/pages/RepoPage.js

```diff
@@ -27,13 +29,17 @@ export const RepoPage = ({ match }) => {
     }
   }, []);
 
+  const commits = (commitHistory[user] && commitHistory[user][repo]) || [];
+
   return (
     <div>
-      <h1>{repo}</h1>
-      <h1>{user}</h1>
-      {commitHistory[user] && commitHistory[user][repo] ? (
-        <CommitList commits={commitHistory[user][repo]} repo={repo} />
-      ) : null}
+      <Heading fontSize="32px" mb="14px" textAlign="center">
+        {repo} by{' '}
+        <Link as={RouterLink} to={`/user/${user}`}>
+          {user}
+        </Link>
```

Comment:
I'm not sure that people will realize this is a button back, the location doesn't really make sense. I wonder if we should add a button somewhere else, like on the left side of the page? 

The rest looks good tho